src/components/Dashboard/TodoList.vue

```diff
@@ -579,7 +613,9 @@ export default {
             status: filing.header.status || FilingStatus.NEW,
             enabled: Boolean(task.enabled),
             order: task.order,
-            paymentToken: filing.header.paymentToken || null
+            paymentToken: filing.header.paymentToken || null,
+            paymentErrorType: bcolErr,
+            bcolErrObj: bcolObj
```

Comment:
Because we only care about payment error type "BCOL..." at the moment, did you consider merging the 2 new properties into a single one? For instance, "if bcolErrObj isn't null then this is a BCOL error". Then you don't need both a flag and the object. Just a thought.

Reply:
done

Reply:
Fixed by refactor.